user_guide.adoc

```diff
@@ -2,8 +2,7 @@
 Kaitai Project
 v0.8
 :toc: left
-:source-highlighter: coderay
-:numbered:
```

Comment:
Why have you removed numberings of chapters?

Reply:
My bad, fixed.